native/ios/Integreat.xcodeproj/project.pbxproj

```diff
@@ -641,6 +641,14 @@
 					" ${PODS_CONFIGURATION_BUILD_DIR}/React-NativeModulesApple/React_NativeModulesApple.framework/Headers",
 					" ${PODS_CONFIGURATION_BUILD_DIR}/React-graphics/React_graphics.framework/Headers",
 					" ${PODS_CONFIGURATION_BUILD_DIR}/React-graphics/React_graphics.framework/Headers/react/renderer/graphics/platform/ios",
+					" ${PODS_CONFIGURATION_BUILD_DIR}/ReactCommon/ReactCommon.framework/Headers",
```

Comment:
why we need this changes?

Reply:
They were autogenerated, I can also revert them

Reply:
ah okay i found them